tests/functional/test_models.py

```diff
@@ -82,6 +87,18 @@ class Employee(models.Model):
         emp = Employee(cv='3addab78-6368-4c55-ac08-22412b6a2a4c')
         self.assertIsInstance(emp.cv, File)
 
+    @patch('pyuploadcare.api_resources.rest_request', autospec=True)
+    def test_not_found_file_uuid(self, request):
+        request.side_effect = InvalidRequestError(404)
+
+        class Employee(models.Model):
+            cv = FileField(validators=[faked_size_validator])
```

Comment:
It is better to place the validator inside the `test_not_found_file_uuid` until it used somewhere else.